pavelib/utils/test/suites/acceptance_suite.py

```diff
@@ -38,10 +38,11 @@ def cmd(self):
 
         cmd = (
             "DEFAULT_STORE={default_store} ./manage.py {system} --settings acceptance harvest --traceback "
-            "--debug-mode --verbosity {verbosity} {report_args} {extra_args}".format(
+            "--debug-mode --verbosity {verbosity} {pdb} {report_args} {extra_args}".format(
```

Comment:
For this you can change, `{verbosity} {pdb} {report_args}` to `{verbosity} {pdb}{report_args}`, along with changing `pdb="--pdb" if self.pdb else ""` to `pdb="--pdb " if self.pdb else ""`